docs/Development Concepts/3.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Protocols Explained by example
+---
+
+![Overview of how protocols interact in Waku v2.](./overview.png)
```

Comment:
Did you generate the diagram yourself? Isntead, install mermaid so you can directly commit the diagram code: https://docusaurus.io/docs/markdown-features/diagrams

Reply:
Will review this in more details once the diagram code is checked in.

Also feel free to remove this for now and we can have a dedicated PR for that.

Reply:
I picked the diagram from the [RFC](https://rfc.vac.dev/spec/10/#overview-of-protocol-interaction) as a PNG itself